src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -72,6 +72,8 @@ public static class EnvironmentSettingNames
         public const string FunctionsPlatformConfigFilePath = "FUNCTIONS_PLATFORM_CONFIG_FILE_PATH";
         public const string TargetBaseScalingEnabled = "TARGET_BASED_SCALING_ENABLED";
         public const string WebsiteNodeDefaultVersion = "WEBSITE_NODE_DEFAULT_VERSION";
+        public const string FunctionsMetricsPublishPath = "FUNCTIONS_METRICS_PUBLISH_PATH";
+        public const string FunctionsAlwaysReadyInstance = "FUNCTIONS_ALWAYSREADY_INSTANCE";
```

Comment:
@balag0 This FUNCTIONS_ALWAYSREADY_INSTANCE env var will need to be in the container start context for Always Ready instances as discussed.

Reply:
We might also end up making this more generic like
InstanceType = "AlwaysReady" or "HTTP" or ..

but this is good for now. we can readjust later if needed

Reply:
To generalize to a setting like FUNCTIONS_INSTANCE_TYPE I'd have to be confident that we actually will have more than two different distinct types. For the example you gave, it might be fine for there to just be a corresponding FUNCTION_HTTP_INSTANCE setting to capture that scenario. Or maybe no setting is needed at all for "http only" and that's transferred via function include list.